internal/controllers/reconciler_test.go

```diff
@@ -1795,6 +1795,66 @@ func (c *controllerTest) commonTests() {
 				t.expectCertificates()
 			})
 		})
+		Context("with modified certificates", func() {
```

Comment:
I suppose this case also cover the previous test context `with a modified CA certificate`?

Reply:
I think it is effectively the same right now, but I thought I may as well retain the two different contexts since the cert.Spec.CommonNames differ in the setup - even though the execution and assertions are the same.

Reply:
Oh okie, sounds good!